roles/base/tasks/user.yml

```diff
@@ -3,17 +3,20 @@
   group:
     name: "{{ base_user.name }}"
     state: present
+  when: system_type in base_user.system_types
```

Comment:
Instead of adding this check to all tasks of `user.yml` you should be able to check for it `main.yml` already I think.

Reply:
fixed in latest update 👍